apps/src/lib/ui/CheckedRadioButton.jsx

```diff
@@ -0,0 +1,34 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import './checkedRadioButton.css';
+
+export class CheckRadioButton extends Component {
+  static propTypes = {
+    id: PropTypes.string,
+    disabledMode: PropTypes.bool,
+    checked: PropTypes.bool.isRequired,
+    value: PropTypes.string.isRequired,
+    onRadioButtonChange: PropTypes.func.isRequired
+  };
+
+  handleChange = event => {
+    this.props.onRadioButtonChange(event.target.value);
+  };
+
+  render() {
+    return (
+      <div>
+        <input
+          type={'radio'}
```

Comment:
nit: "radio" can be a string instead of a javascript object:

```suggestion
          type="radio"
```